test/tests/userobjects/property_read_file/tests

```diff
@@ -0,0 +1,32 @@
+[Tests]
+  issues = '#28190'
+  design = 'PropertyReadFile.md'
+  # See test/tests/functions/piecewise_constant_from_csv for additional tests
+  [csv_read]
+    requirement = 'The system shall include a file reader for data stored in a CSV format'
+    [multiple_files]
+      type = 'Exodiff'
+      input = 'piecewise_constant_elem_multiple.i'
+      exodiff = 'piecewise_constant_elem_multiple_out.e'
+      detail = "with multiple CSV files read on every execution of the user object, "
+               "with the names of each files stored in a separate CSV file."
```

Comment:
```suggestion
      detail = "with the name of each CSV data file to read stored in a separate "
                  "CSV file. At each user object execution, the next data file is read"
```
My attempt at making this more clear. Feel free to modify. The current verbage implies that more than one csv file is read for each execution, which is not the case.